modules/installation-configure-proxy.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// TODO: gcp
+
+// Questions:
+// - Can they provide multiple certs in additionalTrustBundle?
```

Comment:
Yes and would be typical. Most proxy's will be issued an id cert that a) was not signed by an authority from the RHCOS trust bundle and b) that is not issued by a CA, but instead an intermediate CA.